src/views/components-default/TheDefaultMobileNavigation.vue

```diff
@@ -200,14 +194,11 @@ export default {
 <style lang="scss">
 .mew-component--landing-page-menu-mobile {
   .mobile-menu-button .v-icon.v-icon {
-    font-size: 37px;
+    font-size: 43px;
```

Comment:
You can make this default size in the AppBtnMenu, to make is consistent for all

Reply:
Fixed